apps/system/js/quick_settings.js

```diff
@@ -236,7 +236,8 @@ var QuickSettings = {
 
           case this.bluetooth:
             // do nothing if bluetooth isn't ready
-            if (this.bluetooth.dataset.initializing)
+            if (this.bluetooth.dataset.initializing 
+                || this.airplaneMode.dataset.enabling)
```

Comment:
Access to dataset is costly. I would suggest to keep the current airplane mode status on `QuickSettings`.